### DISCUSSION TITLE
Grid: Island detected (149)

### DISCUSSION THREAD

**Discussion by radek-senfeld** [tbnobody/OpenDTU#2140] [Q&A]
Hello all,
does anybody know what is the exact reason for the inverters to report error "Grid: Island detected (149)"?

My setup is 6x Hoymiles HM-800 connected the output of a Victron Multiplus 5000 GX in an island.

I seems to me that every time the production of Hoymiles inverters is roughly equal to the connected AC loads, the Multiplus misses to produce a few sine waves as there's nothing to do for it at that moment. It then causes, I guess, Hoymiles to disconnect as they don't see the Grid.

Anyone having the same observation or any advice?

**Comment by stefan123t** [comment]
Just a wild guess: can you reproduce the issue with an odd number of HM-800, ie eg only five ?

**Reply by radek-senfeld** [reply]
Hi, yes, it's happening with an odd count as well.

edit: Even one inverter running is still reporting this error. I just turned the others off. Should I try to isolate them electrically?

**Comment by a2a4net** [comment]
Hi,  Did you somehow solve this? I have had the same problem for a long time in island mode on Multiplus

**Reply by radek-senfeld** [reply]
Actually, I hope so!

As of now I'm running two Multis 5000/48 in parallel and.. _(knocked on the wood)_

My initial theory seems kinda correct.

The current setup regarding Hoymiles is: 5x Hoymiles HM-800-2T and 4x HM-1500-4T.

My current strategy is to keep Hoymiles inverters slightly below the base consumption or far above but even a very close consumption / production seems.. _(knocked again)_

Side note: There's a noticeable flicker on an old-school light bulb in one of our fridges during the sunset. It might happen generally when the consumption / production is close. Need to observe it more. It's also nothing new. It was the same with one Multi.

Side note #2: In the past it never happened when the AC-in has been connected to a grid and the Multi has been charging. The grid provided the needed stability, I assume.

**Reply by stefan123t** [reply]
@radek-senfeld the original Hoymiles DTU also has a zero-export function. 
That function only optimizes the Power to Consumption gap in 30W steps up/down.
Maybe that can be used for controlling the inverters in your case too ?

Especially during sunset times, we have seen strange Alarm / Error codes (Over/Undervoltage) being reported sometimes by Hoymiles inverters.
As the inverter receives barely enough energy to maintain the RF module communicating, there may be issues with regulating the output power. Especially during cloudy / shady conditions during dusk / dawn.
